Source/WebCore/html/DOMTokenList.h

```diff
@@ -67,9 +67,9 @@ class DOMTokenList final {
     const Vector<AtomString, 1>& tokens() const { return const_cast<DOMTokenList&>(*this).tokens(); }
 
     static ExceptionOr<void> validateToken(StringView);
-    static ExceptionOr<void> validateTokens(const AtomString* tokens, size_t length);
-    ExceptionOr<void> addInternal(const AtomString* tokens, size_t length);
-    ExceptionOr<void> removeInternal(const AtomString* tokens, size_t length);
+    static ExceptionOr<void> validateTokens(std::span<const AtomString> tokens);
```

Comment:
I think I would probably have implemented these as template functions taking concept Container<const AtomString> rather than std::span directly, given these are private. 

The benefit would be that std::span can still be used with them, but also the FixedVector could be used directly as well.

Reply:
I'm passing a Vector at call sites just fine. Vectors get implicitly converted to span just fine for e.g.